platform/osx/os_osx.mm

```diff
@@ -1237,7 +1244,7 @@ static void displays_arrangement_changed(CGDirectDisplayID display_id, CGDisplay
 	ERR_FAIL_COND_V(window_object == nil, ERR_UNAVAILABLE);
 
 	window_view = [[GodotContentView alloc] init];
-	if (@available(macOS 10.14, *)) {
+	if (floor(NSAppKitVersionNumber) >= NSAppKitVersionNumber10_14) {
```

Comment:
What's the point of flooring it? If the number is 1671.5 then the floor is 1671 which doesn't change the outcome of `>=`.

Reply:
Copy-pasted from [os_osx.mm#L1475](https://github.com/godotengine/godot/blob/master/platform/osx/os_osx.mm#L1475) which was copy-pasted from [os_osx.mm#L2178](https://github.com/godotengine/godot/blob/master/platform/osx/os_osx.mm#L2178), or similar line with `<`.